src/state/slices/postsSlice.ts

```diff
@@ -29,9 +29,13 @@ export const postSlice = createSlice({
       });
       return newPosts;
     },
+    appendPost: (state, action: PayloadAction<Post>) => {
+      state.push(action.payload);
```

Comment:
This appends to the end of the array, but I think we want to append new posts to the front of the array so they appear first on the user's timeline. We should use something like `.unshift()` instead, which appends to the front of an array

We'll take a performance hit for sure since appending to the front of an array means you have to copy all of the items in the array back again, but I can't think of a way around it. It'd be super annoying to have to scroll to the bottom of your feed to see your new post, so I think I'm fine with taking the performance nerf

Reply:
Agree, if there are any major performance spikes then we can look at it at a later date.

For now, this has been implemented

Reply:
Nice! And yeah you're right, we'll have to fix the query to use `ORDER BY timestamp DESC`. I think Aninda threw that in one of his profile page PRs so I was thinking we could merge it in at the same time. If not I'll make a new ticket for it